components/BaseRadioButton/styles.js

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components';
+
+export const Radio = styled.label`
```

Comment:
Qué te parece usar una forma más functional y dejarlo así:
```suggestion
export const Radio = styled('label')`
```
Sé que técnicamente son lo mismo, pero pasar de styled.label a styled(Label) es otro modelo mental. Solo una recomendación.